src/traces/contour/hover.js

```diff
@@ -0,0 +1,17 @@
+/**
+* Copyright 2012-2015, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+'use strict';
+
+var heatmapHoverPoints = require('../heatmap/hover');
```

Comment:
One example of the new pattern:

Here, a contour step requires-in only the part of `heatmap` it need. 

This is important because, browserify bundles everything inside a required module i.e. it doesn't parse through it to check which parts of a module is used.